content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -222,9 +222,19 @@ kubectl describe pvc <pvc_name>
 ```
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. Since this is a alpha feature
+both  `ExpandInUsePersistentVolumes` and `ExpandPersistentVolumes` feature gates must be enabled.
 
 #### Resizing an in-use PersistentVolumeClaim
 
+{{< feature-state for_k8s_version="v1.12" state="alpha" >}}
+Expanding in-use PVCs for FlexVolumes is an alpha feature. To enable this feature use  `ExpandInUsePersistentVolumes` 
+and `ExpandPersistentVolumes` feature gates. The FlexVolume plugin uses RequiresFSResize to identify and resize the 
+Persistent Volume. The underlying driver must support resize, if not an error will be returned and Persistent Volume 
+object will not change and the expand request will fail. FlexVolume driver must implement the method `ExpandFS` defined
+in the interface `FSResizableVolumePlugin` along with `ExpandableVolumePlugin` interface and its method `ExpandVolumeDevice`.
```

Comment:
@brahmaroutu couple changes

- The flex driver should implement `ExpandableVolumePlugin` interface **only** if it is to be installed on the controller/master
- The flex driver should implement any volume expansion along with FS resize in `ExpandFS` method if the flex driver will be installed only on the nodes and _not_ on the controller/master